tests/integration/test_pairwise_significance.py

```diff
@@ -324,3 +360,325 @@ def test_mr_x_mr_pairwise_t_tests(self):
         np.testing.assert_array_almost_equal(
             actual.p_vals, load_python_expression("mr-x-mr-pw-pvals")
         )
+        # test new api also
+        np.testing.assert_array_almost_equal(
+            slice_.pairwise_significance_t_stats[1],
+            load_python_expression("mr-x-mr-pw-tstats"),
+        )
+        np.testing.assert_array_almost_equal(
+            slice_.pairwise_significance_p_vals[1],
+            load_python_expression("mr-x-mr-pw-pvals"),
+        )
+
+
+class TestOverlapsCounts(TestCase):
+    def test_overlaps_cat_x_mr_sub_x_mr_sel(self):
+        matrix = Cube(OL.CAT_X_MR_SUB_X_MR_SEL).partitions[0]
```

Comment:
this is not a `matrix` this is `slice` object. It seems weird that you are instanciate a partition and use it to get the assembler. and then having no assertion on the partition but using it only to get the overalps. I mean you are starting from the top-level interface (the user interface) to query the number in the assembler. I'm a little bit concerned about the test approach. Sorry

Reply:
refactored and moved to another module